v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/DataStreamToSpannerShardedMigrationWithMigrationShardIdColumnIT.java

```diff
@@ -242,7 +242,7 @@ public void multiShardMigration() {
             .build();
     result =
         pipelineOperator()
-            .waitForCondition(createConfig(jobInfo1, Duration.ofMinutes(10)), rowsConditionCheck);
```

Comment:
Is this intentional? Both line 235 and this now wait for jobInfo2 to finish.

Reply:
Yes it's intentional, this waits for rowsConditionCheck too